hal_st/std_periph/std_periph_stm32f0xx/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 add_library(hal_st.std_periph_stm32f0xx EXCLUDE_FROM_ALL STATIC)
 install(TARGETS hal_st.std_periph_stm32f0xx EXPORT halStTargets)
 
-target_include_directories(hal_st.std_periph_stm32f0xx PUBLIC
+target_include_directories(hal_st.std_periph_stm32f0xx SYSTEM PUBLIC
```

Comment:
Why add these folders to the system include folders, why not include it as a regular folder?

Reply:
This way warnings in those headers are not considered as breaking for the build.